translationtable/GLOBAL_TERMS.yaml

```diff
@@ -58,7 +58,8 @@
 "Microcell hybrid": "CLO:0036939",
 "Chorionic villus-derived cell line": "CLO:0036940",
 # "Data Catalog Vocabulary": "dcat:!!!"
-"distribution": "dcat:Distribution",
+"distribution": "dcat:distribution",
```

Comment:
Thanks @nicholsn 

We would not want both.
The key is intended to be (move towards) the term's one proper label,

We endevor to keep the dict invertable to find labels from terms as well.

Reply:
Thanks for the response, @TomConlin! 

I was coming at this from the perspective that the generated dataset turtle documents produced by dipper use `dcat:Distribution` as both an object property and a type, whereas these URIs actual have different meanings in the dcat standard:

> [dcat:Distribution](https://www.w3.org/TR/vocab-dcat-2/#Class:Distribution): A specific representation of a dataset. A dataset might be available in multiple serializations that may differ in various ways, including natural language, media-type or format, schematic organization, temporal and spatial resolution, level of detail or profiles (which might specify any or all of the above).

> [dcat:distribution](https://www.w3.org/TR/vocab-dcat-2/#Property:dataset_distribution): An available distribution of the dataset.

Can you help me better understand what you mean by "the term's one proper label"? How do you handle having the correct RDF serialization in this case?

Reply:
the curie-prefix  `dcat:` has a single base-uri (namespace) mapping here:

https://github.com/monarch-initiative/dipper/blob/master/dipper/curie_map.yaml#L25

which is as they recommend 
`http://www.w3.org/ns/dcat#`

yielding
https://www.w3.org/ns/dcat#Distribution
and
https://www.w3.org/ns/dcat#distribution

Both resolving to the same page 
with no directly relevant information....

I guess it will work out.
I had thought you were accommodating
different representations of the same term.. 

but I still  find relying on terms differing only by 
a single letter case 
to be unfortunate and expect it will be fragile.

Reply:
Good point... with dcat 1.0 the redirect would take you to the #ref of the specific term in the spec... looks like with v2 they don't have the redirect, but at least content negotiation is working `curl -H 'Accept:text/turtle' https://www.w3.org/ns/dcat#distribution`

And I agree the whole CamelCase for classes and snakeCase for properties can be brittle compared to obo-style accession ids.

Anyhow, thanks for the merge! Really great tool and datasets that you all are making available. Hoping to learn more about the whole biolink\monarch ecosystem

Reply:
Thank you for the PR!